lib/datadog/core/configuration/settings.rb

```diff
@@ -618,6 +618,27 @@ def initialize(*_)
           # @default `{}`
           # @return [Hash,nil]
           option :writer_options, default: ->(_i) { {} }, lazy: true
+
+          # Client IP configuration
+          # @public_api
+          settings :client_ip do
+            # Whether client IP collection is disabled. This disables client IPs from HTTP requests to be reported in traces.
+            #
+            # @default `DD_TRACE_CLIENT_IP_HEADER_DISABLED` environment variable, otherwise `false`.
+            # @return [Boolean]
+            option :disabled do |o|
+              o.default { env_to_bool(Tracing::Configuration::Ext::ClientIp::ENV_DISABLED, false) }
+              o.lazy
+            end
+            # An optional name of a custom header to resolve the client IP from.
```

Comment:
```suggestion

            # An optional name of a custom header to resolve the client IP from.
```

Linter should have caught this?

Reply:
Whoops.
Rubocop seems to pass but I shouldv'e considered the formatting in the other options.
Fixed